kornia/augmentation/augmentation.py

```diff
@@ -95,13 +98,13 @@ def __init__(self, p: float = 0.5, return_transform: bool = False) -> None:
         self.p: float = p
         self._params: Dict[str, torch.Tensor] = {}
 
-    def set_params(self, batch_size: int, p: float):
-        self._params = pg._random_prob_gen(batch_size, p)
+    def get_params(self, batch_size: int) -> Dict[str, torch.Tensor]:
```

Comment:
@shijianjian maybe kwargs can be used here to make all calls standard ? @anguelos had some suggestions about it yesterday